src/60_AMP_Ads/Introduction.html

```diff
@@ -0,0 +1,51 @@
+<!---{
+ "preview": "a4a",
+ "skipValidation": "true"
+}--->
+<!--
+  ## Introduction
+
+  [AMP Ads](https://github.com/ampproject/amphtml/blob/master/extensions/amp-a4a/amp-a4a-format.md) are based on the AMP runtimne and provide the same benefits as normal AMPHTML files. However, there are a few things that make them different from normal AMPHTML files. 
```

Comment:
ok - then we should change the amp4ad template here: https://validator.ampproject.org/#htmlFormat=AMP4ADS

Also, the validator doesn't complain when using the normal amp runtime. Is this intended?